File: VB/Demos/11Json/Person.vb
Imports System.Text.Json.Serialization

Namespace Json
  Public Class Person
    Public Property Id As Guid
    Public Property FirstName As String
    Public Property LastName As String
    Public Property Number As Integer
    Public Property Address As String
    Public Property Place As String
    Public Property State As String
    Public Property Zip As String
    <JsonIgnore>
    Public Property NonSerializedData As String
  End Class
End Namespace

File: CS/Demos/11Json/Person.cs
using System.Text.Json.Serialization;

namespace Demos.Json;

class Person
{
  public Guid Id { get; set; }
  public string? FirstName { get; set; }
  public string? LastName { get; set; }
  public int Number { get; set; }
  public string? Address { get; set; }
  public string? Place { get; set; }
  public string? State { get; set; }
  public string? Zip { get; set; }
  [JsonIgnore]
  public string? NonSerializedData { get; set; }
}
